### PR TITLE
Homepage update

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)
Update homepage with webhint logo and content updates

### CODE REVIEW

File: src/webhint-theme/layout/partials/home.hbs [hunk 1]
```diff
@@ -59,18 +59,39 @@
...
+                <div class="layout layout--halves">
                     <div class="module">
-                        <img src="images/cli.gif" alt="webhint cli">
+                        <img src="images/hint.gif" alt="webhint cli">
                     </div>
                     <div class="module">
                         <p class="install__code">If you want to disable specific hints, further configure others, ignore domains, or more, run webhint from your CLI.</p>
                         <p class="install__code">Quick start guide:</p>
                         <div class="code-wrap">
                             <p>><code>npm install -g --engine-strict webhint</code></p>
```
Comment (sarvaje) [inline src/webhint-theme/layout/partials/home.hbs]: This should be `npm install -g --engine-strict hint`

Comment (molant) [reply src/webhint-theme/layout/partials/home.hbs]: Maybe we should have the npx command here now that it works? @webhintio/core?

**Comment by ststimac** [comment]
Updated CLI instructions.

File: src/webhint-theme/layout/partials/home.hbs [hunk 2]
```diff
@@ -59,18 +59,39 @@
...
                     <div class="module">
-                        <img src="images/cli.gif" alt="webhint cli">
+                        <img src="images/hint.gif" alt="webhint cli">
                     </div>
                     <div class="module">
                         <p class="install__code">If you want to disable specific hints, further configure others, ignore domains, or more, run webhint from your CLI.</p>
                         <p class="install__code">Quick start guide:</p>
                         <div class="code-wrap">
-                            <p>><code>npm install -g --engine-strict webhint</code></p>
+                            <p>><code>npx install -g --engine-strict hint</code></p>
```
Comment (sarvaje) [inline src/webhint-theme/layout/partials/home.hbs]: I think the command is `npx hint https://awesomesite.com` and the next two lines are not necessary. @alrra am I right or wrong?

Comment (alrra) [reply src/webhint-theme/layout/partials/home.hbs]: > I think the command is npx hint https://awesomesite.com

Yes.

>  the next two lines are not necessary. 

Yes.

Comment (ststimac) [reply src/webhint-theme/layout/partials/home.hbs]: So just to clarify, I only need the line:

npx hint https://awesomesite.com

Comment (alrra) [reply src/webhint-theme/layout/partials/home.hbs]: Yes.

**Comment by ststimac** [comment]
OK, updated again to get rid of the other two lines in the CLI.

File: src/webhint-theme/source/images/webhint-logo.svg [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	c-0.1,0-0.1,0-0.2,0h-0.9c-0.5,0-1-0.3-1-0.9C152.2,32.2,152.2,32.1,152.3,32.1L152.3,32.1z M161.3,32.1V15.4c0-0.5,0.3-1,0.9-1
+	c0,0,0.1,0,0.1,0h0.9c0.5,0,1,0.3,1,0.9c0,0,0,0.1,0,0.1v3c1-2.7,3.6-4.5,6.5-4.4c4,0,7.4,2.3,7.4,8.5v9.6c0,0.5-0.3,1-0.9,1
+	c0,0-0.1,0-0.1,0h-0.9c-0.5,0-1-0.3-1-0.9c0,0,0-0.1,0-0.1v-9.7c0-4.5-2.4-6-4.7-6c-2.1,0-6.3,1.5-6.3,7.5v8.1c0,0.5-0.3,1-0.9,1
+	c0,0-0.1,0-0.1,0h-0.9c-0.5,0.1-1-0.3-1-0.8C161.3,32.2,161.3,32.2,161.3,32.1L161.3,32.1z M184.5,27.2V16.7h-2.6
+	c-0.4,0-0.6-0.2-0.6-0.6V15c0-0.3,0.2-0.6,0.4-0.6c0.1,0,0.1,0,0.2,0h2.6l0.6-4.8c0-0.5,0.4-0.9,0.9-0.8c0,0,0,0,0,0h0.5
+	c0.4-0.1,0.8,0.3,0.9,0.7c0,0.1,0,0.1,0,0.2v4.8h5.4c0.3,0,0.6,0.2,0.6,0.5c0,0,0,0.1,0,0.1v1.1c0,0.3-0.2,0.6-0.5,0.6
+	c0,0-0.1,0-0.1,0h-5.4v10.8c0,2.7,1.3,3.6,2.7,3.6c1.4,0.1,2.5-0.9,2.7-2.2c0-0.1,0-0.2,0-0.2c0-0.2,0-0.5-0.1-0.7
+	c0-0.5,0.1-0.7,0.5-0.7h1.3c0.3,0,0.6,0.2,0.7,0.5c0.1,0.4,0.1,0.8,0.1,1.2c0,2.6-1.8,4.6-5.3,4.6C187.2,33.5,184.5,32.2,184.5,27.2
+	L184.5,27.2z"/>
+</svg>
```
Comment (alrra) [inline src/webhint-theme/source/images/webhint-logo.svg]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 202 39.5"><style>.a{fill:#fff}</style><path d="M19.8 0C8.8 0 0 8.8 0 19.8s8.8 19.8 19.8 19.8 19.8-8.8 19.8-19.8C39.5 8.8 30.7 0 19.8 0zm17.4 19.8c0 9.6-7.8 17.5-17.5 17.5S2.3 29.4 2.3 19.7c0-9.6 7.8-17.5 17.5-17.5 9.6.1 17.4 7.9 17.4 17.6z" class="a"/><path d="M29.4 15.2L30 14l1.9-3.1-4.4 2.6c-.1.1-.2.1-.3.2-.5.5-.5 1.4 0 1.9.2.2.4.3.6.3.5.1 1 0 1.3-.3.1-.1.2-.2.3-.4z" class="a"/><path d="M28.8 16.9c-.3.1-.5.1-.8.1-1.2 0-2.2-1-2.2-2.2 0-.2 0-.5.1-.7-1.1-.7-2.4-1-3.6-.9-6.8 0-7.9 3.2-7.9 7.2.1 1 0 2-.5 2.9-.5.9-1.5 1.5-2.5 1.4-.4 0-1.9-.3-2-.7h.2c1.6 0 2.9-1.3 2.9-3v-.8h-1.4c-1 0-1.9.5-2.5 1.3C8 20.8 7 20.3 6 20.3H4.6v.6C4.6 22.6 6 24 7.8 24h.5c0 1 1.4 5 11.7 5 4 0 10-.7 10-8.3-.1-1.3-.5-2.7-1.2-3.8zm28.6 15.4l-6.3-16.7c-.3-.7.1-1.2.8-1.2h.9c.5-.1 1 .3 1.2.8l5.2 14.1 4.5-13.4c.1-.5.6-.8 1.1-.8h.8c.5 0 1 .3 1.1.8l4.5 13.4 5.2-14c.1-.5.6-.9 1.1-.8h.9c.8 0 1.1.5.8 1.2l-6.3 16.7c-.1.5-.6.9-1.2.8h-1c-.5 0-1-.3-1.1-.8l-4.5-13.2-4.4 13.2c-.1.5-.6.9-1.1.8h-1c-.6-.1-1.1-.4-1.2-.9zm24.2-8.5c0-5.4 4.3-9.8 9.7-9.9h.3c4.6 0 8.9 2.7 9.3 8.8.1.6-.3 1-1 1H84.5c-.3 4.2 3 7.3 7.6 7.3 2 0 4-.6 5.6-1.9.4-.3 1-.3 1.4.1l.3.3c.4.4.4 1 0 1.4l-.2.2c-2 1.6-4.5 2.5-7.1 2.4-6.4 0-10.5-4.4-10.5-9.7zm16.5-2.3c-.5-3.1-3.3-5.2-6.4-5a7 7 0 0 0-7 5h13.4zm10.1 7.9v2.7c0 .5-.3 1-.9 1h-1c-.5 0-1-.3-1-.8V7c0-.5.3-1 .8-1h1.1c.5 0 1 .3 1 .8V18c1.5-2.6 4.3-4.2 7.3-4.1 5.7 0 9.4 4.4 9.4 9.8s-3.7 9.8-9.4 9.8c-3 .1-5.8-1.5-7.3-4.1zm13.8-5.6c0-4-2.5-7.3-6.8-7.3-3.8-.1-7 2.9-7.1 6.7v.5c-.2 3.8 2.8 7 6.6 7.2h.5c4.3.1 6.8-3.2 6.8-7.1zm7.6 8.3V7c0-.5.3-1 .8-1h1.1c.5 0 1 .3 1 .9v11.5c1-2.7 3.7-4.5 6.6-4.4 4 0 7.4 2.3 7.4 8.5v9.6c0 .5-.3 1-.9 1h-1c-.5 0-1-.3-1-.9v-9.7c0-4.5-2.4-6-4.7-6-2.1 0-6.3 1.5-6.3 7.5v8.1c0 .5-.3 1-.9 1h-1c-.5.1-1-.3-1.1-.8v-.2zm22-24c-.1-1.1.8-2.1 1.9-2.1h.2c1.1 0 2.1.9 2.1 2v.1c0 1.1-.9 2.1-2 2.1h-.1c-1.1 0-2-.8-2.1-1.9v-.2zm.7 24V15.4c0-.5.4-1 .9-1h1c.5-.1.9.3 1 .8v16.9c.1.5-.3.9-.8 1h-1.1c-.5 0-1-.3-1-.9-.1 0-.1-.1 0-.1zm9 0V15.4c0-.5.3-1 .9-1h1c.5 0 1 .3 1 .9v3.1c1-2.7 3.6-4.5 6.5-4.4 4 0 7.4 2.3 7.4 8.5v9.6c0 .5-.3 1-.9 1h-1c-.5 0-1-.3-1-.9V22.4c0-4.5-2.4-6-4.7-6-2.1 0-6.3 1.5-6.3 7.5V32c0 .5-.3 1-.9 1h-1c-.5.1-1-.3-1-.8v-.1zm23.2-4.9V16.7h-2.6c-.4 0-.6-.2-.6-.6V15c0-.3.2-.6.4-.6H184.5l.6-4.8c0-.5.4-.9.9-.8h.5c.4-.1.8.3.9.7v5h5.4c.3 0 .6.2.6.5V16.2c0 .3-.2.6-.5.6h-5.5v10.8c0 2.7 1.3 3.6 2.7 3.6 1.4.1 2.5-.9 2.7-2.2v-.2c0-.2 0-.5-.1-.7 0-.5.1-.7.5-.7h1.3c.3 0 .6.2.7.5.1.4.1.8.1 1.2 0 2.6-1.8 4.6-5.3 4.6-2.8-.2-5.5-1.5-5.5-6.5z" class="a"/></svg>
```

**Pull request closed without merging**